src/main/java/core/basesyntax/dao/ma/MentorDaoImpl.java

```diff
@@ -11,6 +15,14 @@ public MentorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public List<Mentor> findByAgeGreaterThan(int age) {
-        return null;
+        try (Session session = super.sessionFactory.openSession()) {
```

Comment:
does your MentorDaoImpl class have the sessionFactory field?

Reply:
Yes. Fixed! :)